### DISCUSSION TITLE
Is OpticSim.jl still under active development?

### DISCUSSION THREAD

**Discussion by fofusa** [brianguenter/OpticSim.jl#32] [General]
Hello,
I played a bit with OpticSim.jl a couple of years ago (when it was hosted on github.com/microsoft).
I am considering using it more seriously in the coming years but I wonder whether it is still under active development as I don't see much activity on this repo (not a criticism, just an honest question)?
I have experienced a certain number of issues, but I don't know whether I should open them here or how to contribute.

Cordially,

Fofusa

**Comment by brianguenter** [comment]
OpticSim has not been under active development for 18 months or so.

I am considering actively developing it again but that will be at least
several months in the future.

If it wouldn't be too much trouble go ahead and raise an issue for your
problems. I'll take a look and see if it's something I can easily fix or
explain.

-brian

-brian

**Comment by fofusa** [comment]
Hello Brian,

Thanks for your answer.
I've mainly noticed issues with the doc or missing glass catalog or examples that cannot run out of the box.
I'll try to raise some issues and I'll be glad to contribute (not sure I'm fluent enough in julia to contribute anything useful).

Fofusa.
